docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,164 @@
+##########################
+Add a Zone Entity
+##########################
+
+A zone entity allows you to set properties to a specific area. These properties include: flying, ghosting, key lighting, skybox, ambient lighting, hase, bloom and avatar priorities.
+
+.. contents:: On This Page
+    :depth: 3
+
+-------------------------
+Explanation of Properties
+-------------------------
+
+The amount of properties might seem intimidating at first. They all serve their own goals and have reason to exist:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Flying Allowed         | Enables or disables flying for people inside the zone. This makes sense for gamemodes, or   |
+|                        | if you just don't want people to break your immersion.                                      |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ghosting Allowed       | Enables or disables the ability to phase through colliders by disabling the enviroment      |
+|                        | collisions. Useful for making labyrinths for example.                                       |
++------------------------+---------------------------------------------------------------------------------------------+
+| Filter                 | Filters are specialized JavaScript functions that prevent unwanted modifications to         |
+|                        | entities inside the zone. This property takes the URL to such a script.                     |
+|                        | More information here:                                                                      |
+|                        | `Protect Your Domain’s Content <../../host/configure-settings/entity-filters.html>`_        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Key Light              | Enabling this allows you to set a directional light, e.g. a sun, for the entire zone.       |
++------------------------+---------------------------------------------------------------------------------------------+
+| Skybox                 | This allowed you to set a so called "Skybox". This is commonly used to apply a sky texture  |
+|                        | the zone.                                                                                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ambient Light          | This is used to apply light coming from the Skybox. An example usecase for this would be to |
```

Comment:
Isn't ambient light for general, environmental light; such as may come from the environment in general, not a particular light source? And to set the light coming from a sun on the skybox you would typically use the key light property?

Reply:
According to the tooltips in the create menu, and my tests, it uses the texture as a lightsource.
The example with the sun might not be the best idea.
Further down is a more fitting example `One scenario where this could be useful, would be a spacestation, that is illuminated my multiple suns and reflections from different colored planets.`

Reply:
Perhaps a sunset with colored clouds in the sky would be a better example? (Illustrates light coming from multiple directions rather than a point source.)

Reply:
Changed this to `This is used if you have light coming from more than just a sun. For example if you have multiple suns.` for now, since I don't think clouds in sunset are a very clear example either.